src/Files.Uwp/UserControls/Widgets/DrivesWidget.xaml

```diff
@@ -1,237 +1,251 @@
 ﻿<UserControl
-    x:Class="Files.UserControls.Widgets.DrivesWidget"
-    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
-    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-    xmlns:navigationcontrolitems="using:Files.DataModels.NavigationControlItems"
-    xmlns:helpers="using:Files.Helpers"
-    xmlns:local="using:Files.UserControls.Widgets"
-    xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
-    Background="{ThemeResource SolidBackgroundFillColorBaseBrush}"
-    mc:Ignorable="d">
-    <Grid>
-        <Grid>
-            <Grid.ColumnDefinitions>
-                <ColumnDefinition />
-                <ColumnDefinition />
-            </Grid.ColumnDefinitions>
+	x:Class="Files.UserControls.Widgets.DrivesWidget"
+	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	xmlns:local="using:Files.UserControls.Widgets"
+	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
+	xmlns:helpers="using:Files.Helpers"
+	Background="{ThemeResource SolidBackgroundFillColorBaseBrush}"
+	mc:Ignorable="d">
+	<Grid>
+		<Grid>
+			<Grid.ColumnDefinitions>
+				<ColumnDefinition />
+				<ColumnDefinition
+					Width="Auto" />
+			</Grid.ColumnDefinitions>
```

Comment:
Can you fix the formatting \?

Reply:
@yaichenbaum No. It looks fine on my end.